auxx/src/Main.hs

```diff
@@ -104,5 +105,10 @@ action opts@AuxxOptions {..} = withConfigurations conf $ do
 main :: IO ()
 main = withCompileInfo $(retrieveCompileTimeInfo) $ do
     opts <- getAuxxOptions
-    let loggingParams = CLI.loggingParams "auxx" (aoCommonNodeArgs opts)
-    loggerBracket loggingParams $ runProduction $ action opts
+    let loggingParams = (CLI.loggingParams "auxx" (aoCommonNodeArgs opts))
+                        { lpConsoleLog = Just False }
```

Comment:
I think console log should be disabled only for repl, am I wrong?

Reply:
The node tries to print to `stdout`, it disrupts the REPL.

Reply:
You probably got me wrong, I mean that if `aoAction opts` is `Cmd`, do we want to disable console output?

Reply:
ah, I see. Alright, fixed.